src/persist-types/src/parquet.rs

```diff
@@ -31,61 +33,48 @@ use crate::part::{Part, PartBuilder};
 ///
 /// It doesn't particularly get any anything to use more than one "chunk" per
 /// blob, and it's simpler to only have one, so do that.
-pub fn encode_part<W: Write>(w: &mut W, part: &Part) -> Result<(), anyhow::Error> {
-    let metadata = Vec::new();
-    let (fields, encodings, chunk) = part.to_arrow();
-
-    let schema = ArrowSchema::from(fields);
-    // Construct a FileWriter manually so we can omit the created_by string and
-    // (redundant) arrow schema.
-    let parquet_schema = to_parquet_schema(&schema)?;
-    let options = WriteOptions {
-        write_statistics: false,
-        compression: CompressionOptions::Uncompressed,
-        version: Version::V2,
-        data_pagesize_limit: None, // use default limit
-    };
-    let created_by = None;
-    let mut writer = FileWriter::new(
-        w,
-        parquet_schema.clone(),
-        ParquetWriteOptions {
-            version: options.version,
-            write_statistics: options.write_statistics,
-        },
-        created_by,
-    );
-
-    let row_group = DynIter::new(row_group_iter(
-        chunk,
-        encodings,
-        parquet_schema.fields().to_vec(),
-        options,
-    ));
-    writer.write(row_group)?;
-    writer.end(Some(metadata))?;
+pub fn encode_part<W: Write + Send>(w: &mut W, part: &Part) -> Result<(), anyhow::Error> {
+    let (fields, arrays) = part.to_arrow();
+
+    let schema = Arc::new(ArrowSchema::new(fields));
+    let props = WriterProperties::builder()
+        .set_dictionary_enabled(false)
+        .set_encoding(Encoding::PLAIN)
+        .set_statistics_enabled(EnabledStatistics::None)
+        .set_compression(Compression::UNCOMPRESSED)
+        .set_writer_version(WriterVersion::PARQUET_2_0)
+        .set_data_page_size_limit(1024 * 1024)
```

Comment:
does this match the default size previously set by arrow2? I wonder if this would have any encode/decode perf impact if different.

Reply:
> does this match the default size previously set by arrow2?
Exactly!

It probably has some perf impact, but at the moment wanted to keep things 1:1 with `arrow2`